### PR TITLE
New persondetails view

### PR DESCRIPTION
**Pull request by sevenrats**
**Changes**

muck about with the layout to make better use of space.
increase text size in dialog
always allow dialog selection even if the text is not ellipsized
set focus to description by default instead of fav menu, assuming that the use pattern will be to inspect the description before favoriting.

react to missing birthdate info by increasing Name size to elliminate deadspace. 

- [x] unbreak the overview dialog

### CODE REVIEW

**Review by candry7731 — APPROVED** [review]
Like the layout with moving the favorite button to the top. Also styling the person details makes this a great addition. 

it would be great that if the person has no info we show "no info" or something like that instead of a blank box... something like below.

image

**Comment by sevenrats** [comment]
I agree.  also need to balance the margins relative to the slider.

**Comment by candry7731** [comment]
Much better with the "no details" options. 

I am seeing the vert align when clicking on a show then viewing the extra's info however it's missing when clicking from a person then seeing there extra's info... 
image

I'm also getting this error
image

**Comment by sevenrats** [comment]
yeah the code is filthy dirty right now. The main things I did were change the font size if birthdate info is missing, and move the layout around when we have it to try and maximize the balance with the submit button.  I fixed the slider, I think.

**Comment by candry7731** [comment]
the slider spacing looks good until you go in from a person details screen then click into the slider and the margins revert to being 0 margin on the left side and a hug gap on the right....

File: components/PersonDetails.brs [hunk 1]
```diff
@@ -50,39 +63,30 @@ sub loadPerson()
...
 
 sub onButtonGroupEscaped()
     key = m.btnGrp.escape
     if key = "down"
-        m.vidsList.setFocus(true)
-        m.top.findNode("VertSlider").reverse = false
-        m.top.findNode("pplAnime").control = "start"
-    else if key = "up" and m.dscr.isTextEllipsized
-        dscrShowFocus()
+        print "key down"
```
Comment (1hitsong) [inline components/PersonDetails.brs]: Debug code

File: locale/en_US/translations.ts [hunk 2]
```diff
@@ -1108,5 +1112,9 @@
             <translation>Set how many seconds before the end of an episode the Next Episode button should appear. Set to 0 to disable.</translation>
             <extracomment>Settings Menu - Description for option</extracomment>
         </message>
+        <message>
+            <source>Biographical information for this person is not currently available.</source>
```
Comment (1hitsong) [inline locale/en_US/translations.ts]: Same line of text already added on line 1024.

**Review by 1hitsong — CHANGES_REQUESTED** [review]

**Review by 1hitsong — APPROVED** [review]

**Comment by 1hitsong** [comment]
To be merged 2/12 unless issue arises.

**Pull request merged**